client/styles/components/_console.scss

```diff
@@ -92,12 +92,22 @@
 	}
 	background: transparent;
 	border: none;
+	border-radius: 20px;
+	margin-right: 5px; 
 	padding-right: #{10 / $base-font-size}rem;
 	.preview-console--collapsed & {
 		display: none;
 	}
 }
 
+.preview-console__clear:hover {
+
+	background-color: rgba(45, 38, 38, 0.345);
```

Comment:
We generally don't hard-code colors in the CSS because we have three different themes.  What looks good on dark might not look good on light and vice-versa.  You would want to use a value from the theme with `getThemifyVariable`.  If you need a new color value that's not already defined then you need to add it to all three themes.